cdc/entry/schema/snapshot.go

```diff
@@ -203,60 +215,6 @@ func NewEmptySnapshot(forceReplicate bool) *Snapshot {
 	return &Snapshot{inner: inner, rwlock: new(sync.RWMutex)}
 }
 
-// these constants imitate TiDB's session.InitDDLJobTables in an empty Snapshot.
-const (
-	mysqlDBID      = int64(1)
-	dummyTS        = uint64(1)
-	mdlCreateTable = "create table mysql.tidb_mdl_info(job_id BIGINT NOT NULL PRIMARY KEY, version BIGINT NOT NULL, table_ids text(65535));"
-)
-
-// InitPreExistingTables initializes the pre-existing tables in an empty Snapshot.
-// Since v6.2.0, tables of concurrent DDL will be directly written as meta KV in
-// TiKV, without being written to history DDL jobs. So the Snapshot which is not
-// build from meta needs this method to handle history DDL.
-// Since v6.5.0, Backfill tables is written as meta KV in TiKV, so the Snapshot
-// which is not build from meta needs this method to handle history DDL.
-// See:https://github.com/pingcap/tidb/pull/39616
-func (s *Snapshot) InitPreExistingTables() {
```

Comment:
Could you explain why we remove this?

Reply:
Based on our analysis, it appears that TiCDC does not rely on the pre-existing system table. As such, I recommend removing this function and discontinuing its use.